### ISSUE TITLE
ContextualMenu section headers need to be contained in groups, so the headers could be pointed at via aria-labelledby

### ISSUE THREAD

**Issue by stg110** [OfficeDev/office-ui-fabric-react#6549]
### Please provide a reproduction of the bug in a codepen:
. Turn on Narrator from ease of access setting.
2. Open the Word app using the URL: https://microsoft.sharepoint-df.com/teams/TestSiteCreation/Shared%20Documents/Forms/AllItems.aspx 
with valid credentials.
3. Get the focus on ‘Ribbon’ using ‘Ctrl+F6’ key through keyboard.
4. Navigate on ‘Home’ tab and activate using keyboard.
5. Navigate to ‘Dictate’ Button and activate it using ‘Alt+Down’ arrow key.
6. Navigate on Different Language and listen Narrator.
7. Listen Narrator is announcing the Group Name for ‘Preview Language’ or not.
 
Actual:
'Preview Languages' is not defined as a 'Group Name' for languages so that Narrator is not announcing the group name, when user is navigating on the ‘Languages’ define under ‘Preview Languages’.
 
Expected:
'Preview Languages' should be defined as a 'Group Name' for languages so that Narrator will announce the group name as ‘Preview Languages’, when user is navigating on the ‘Languages’ defined under ‘Preview Languages’.
 

### Bug Report 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low)  Normal
 
Products/sites affected: (if applicable) Word Online Dictation Feature
 
#### Describe the issue:
 
menu sections are not announced as group name when using narrator

https://office.visualstudio.com/OC/_workitems/edit/2588494

**Comment by Vitalius1** [comment]
@KevinTCoughlin FYI as you are listed as the owner of this control.

**Comment by SavannahBourgeois** [comment]
We got an accessibility bug in office-online related to this. Any way we can bump this to be scheduled work?

**Comment by kenotron** [comment]
I dug into the office online code, @SavannahBourgeois - the bug resides in how ContextualMenu is being consumed. I've reduced this bug into this example:

https://codesandbox.io/embed/gallant-sun-p3wqh

You'll find the correct and incorrect way to group items. It looks like office online code generates a flat list of items with some that are styled. Fabric will happily take in section groups and will do the grouping correctly. Another example of working example is the "ContextualMenu with submenus" example here:

https://developer.microsoft.com/en-us/fabric#/controls/web/contextualmenu

I'll ping you about the exact file that needs to be modified internally.

**Comment by kenotron** [comment]
I've marked this bug external and will let bot to close it. OP can close it at the earliest convenience.

**Comment by jspurlin** [comment]
FYI, this has been fixed by https://github.com/OfficeDev/office-ui-fabric-react/commit/978edb71c86f2c4a32411b7b78097a26fb6e9044#diff-570dcaf84c12ad2593f547152a02ba16
